gradle/libs.versions.toml

```diff
@@ -25,6 +26,7 @@ grpc-alts = { module = "io.grpc:grpc-alts", version.ref = "grpc" }
 grpc-netty = { module = "io.grpc:grpc-netty", version.ref = "grpc" }
 
 provenance-protos = { module = "io.provenance:proto-kotlin", version.ref = "provenance-protos" }
+protobuf-kotlin = { module = "com.google.protobuf:protobuf-kotlin", version.ref = "protobuf" }
```

Comment:
I had to add this in to get the kotlin dsl proto syntax in the coroutine client to not blow up